app/src/main/java/com/mobileasone/dagger2workshop/di/NetworkApiFactory.kt

```diff
@@ -1,14 +0,0 @@
-package com.mobileasone.dagger2workshop.di
-
-import com.mobileasone.dagger2workshop.data.network.NotesServiceApi
-import com.mobileasone.dagger2workshop.data.network.NotesServiceApiImpl
-
-object NetworkApiFactory {
-
-    private val notesServiceApiInstance: NotesServiceApi by lazy { initNotesServiceApi() }
-
-    fun createNetworkApi(): NotesServiceApi = notesServiceApiInstance
```

Comment:
Now we can remove NetworkApiFactory since we have the [SampleAppComponent](https://github.com/Jamargle/Dagger2Workshop/pull/5/files#diff-f6c7ac2e2280f5df175fb97e75c929e6) that provides the instance of [NotesServiceApi](https://github.com/Jamargle/Dagger2Workshop/blob/step2-di-with-dagger2/app/src/main/java/com/mobileasone/dagger2workshop/data/network/NotesServiceApiImp.kt) everywhere in the app through dependency injection if we do the setup for it.